### ISSUE TITLE
Hotkey support

### ISSUE THREAD

**Issue by kherchel** [cybre/budgie-screenshot-applet#24]
It would be really nice if it was possible to take screenshots using hotkeys.
Screencloud has such a feature where you can configure separate key combinations for Capture screen, capture window and capture selection, then it automatically copies the URL to clipboard. Super handy.

**Comment by cybre** [comment]
This is not possible with the current Budgie stack. I will leave this issue open so that others can see it before requesting.

**Comment by olback** [comment]
Isn't this possible with Gnome Hot Keys? You can configure your own Hot-key-combos, so all that is needed is a command to trigger a screenshot. I know the Budgie team is trying to "de-gnomeify'" Budgie but it would work a little while at least.

**Comment by cybre** [comment]
This is an applet (a plugin). There is no command/executable that you can
run.

**Comment by zor1984qq** [comment]
Same here. I can't figure out how to do screenshot of main budgie menu with a hotkey cause without it its just dissapearing as I switching to gnome-screenshot tool. I think its better to try to use shutter.

**Comment by mazunki** [comment]
I made a script to automatically press the applet for me, as a workaround, available here:

https://github.com/mazunki/mybash/blob/master/screenshotshortcut.sh

I can't get the keyboard shortcut to work on my Manjaro environment, though, as adding shortcuts to `Settings->Devices->Keyboard Shortcuts` doesn't seem to work. I've `chmod +x screenshotshortcut.sh`'d my file, but no fish. 

(Some) other commands in the list do work, though. I've restarted my session, too, so it should be updated. Is there something I'm missing?
